addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -288,7 +288,7 @@ def migration_invoice_moves(env):
                 THEN ail.currency_id ELSE NULL END, ail.partner_id, ail.uom_id,
                 ail.product_id, ail.account_analytic_id, ail.display_type,
                 ail.is_rounding_line, COALESCE(ai.move_id, am.id), ail.id, ai.date_invoice,
-                ail.create_uid, ail.create_date, ail.write_uid, ail.write_date
+                COALESCE(ai.date, ail.create_uid), ail.create_date, ail.write_uid, ail.write_date
```

Comment:
The coalesce should be in the previous field, not here

Reply:
Yes, sorry, my mistake, for going too fast.